ror-demo-cluster/conf/es/readonlyrest.yml

```
@@ -1,63 +1,199 @@
+      ssl_enabled: false
+      ssl_trust_all_certs: true
+
+      bind_dn: "uid=admin,ou=people,dc=example,dc=com"
+      bind_password: "${env:LDAP_ADMIN_PASSWORD}"
+
+      search_user_base_DN: "ou=People,dc=example,dc=com"
+      search_groups_base_DN: "ou=Groups,dc=example,dc=com"
+      user_id_attribute: "uid"                                  
+      unique_member_attribute: "uniqueMember"
+
+      cache_ttl: 10s
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)